File: src/main/java/ru/ioleksiv/telegram/bot/core/model/objects/payments/OrderInfo.java
package ru.ioleksiv.telegram.bot.core.model.objects.payments;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * @see <a href="https://core.telegram.org/bots/api#orderinfo>OrderInfo</a>
 */
public class OrderInfo {
    /**
     * name	String	Optional. User name
     */
    @JsonProperty("name")
    private String name = null;
    /**
     * phone_number	String	Optional. User's phone number
     */
    @JsonProperty("phone_number")
    private String phoneNumber = null;
    /**
     * email	String	Optional. User email
     */
    @JsonProperty("email")
    private String email = null;
    /**
     * shipping_address	ShippingAddress	Optional. User shipping address
     */
    @JsonProperty("shipping_address")
    private ShippingAddress shippingAddress = null;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public ShippingAddress getShippingAddress() {
        return shippingAddress;
    }

    public void setShippingAddress(ShippingAddress shippingAddress) {
        this.shippingAddress = shippingAddress;
    }
}

File: server-api/src/main/kotlin/io/github/oleksivio/tl/kbot/server/api/objects/payments/OrderInfo.kt
package io.github.oleksivio.tl.kbot.server.api.objects.payments

import com.fasterxml.jackson.annotation.JsonProperty
import io.github.oleksivio.tl.kbot.server.api.model.ITelegram

/**
 *  [OrderInfo](https://core.telegram.org/bots/api/#orderinfo)
 */
data class OrderInfo(
    /**
     * To setup filter:
     *
     *  StringFilter name name [String] Optional. User name
     */
    @JsonProperty("name")
    var name: String? = null,
    /**
     * To setup filter:
     *
     *  StringFilter phoneNumber phone_number [String] Optional. User's phone number
     */
    @JsonProperty("phone_number")
    var phoneNumber: String? = null,
    /**
     * To setup filter:
     *
     *  StringFilter email email [String] Optional. User email
     */
    @JsonProperty("email")
    var email: String? = null,
    /**
     * To setup filter:
     *
     *  ShippingAddressFilter  shippingAddress shipping_address ShippingAddress Optional. User shipping address
     */
    @JsonProperty("shipping_address")
    var shippingAddress: ShippingAddress? = null
) : ITelegram
